packages/devtools-evm-hardhat/src/runtime.ts

```diff
@@ -218,8 +219,28 @@ export const getNetworkNameForEid = (
         }
     }
 
+    return undefined
+}
+
+/**
+ * Gets a network name with its `eid` property matching
+ * a particular `eid`
+ *
+ * Throws if there is no such network or if there are multiple
+ * networks defined with the same `eid`
+ *
+ * @param {EndpointId} eid
+ * @param {HardhatRuntimeEnvironment | undefined} [hre]
+ * @returns {string}
+ */
+export const getNetworkNameForEid = (
+    eid: EndpointId,
+    hre: HardhatRuntimeEnvironment = getDefaultRuntimeEnvironment()
+): string => {
+    const networkName = getNetworkNameForEidMaybe(eid, hre)
+
     // Here we error out if there are no networks with this eid
-    assert(false, `Could not find a network for eid ${eid} (${formatEid(eid)})`)
+    return assert(networkName != null, `Could not find a network for eid ${eid} (${formatEid(eid)})`), networkName
```

Comment:
hmm would network name ever be `null`? Seems like it would only ever be `string | undefined`

Reply:
`a == null` / `a != null` is a very safe way of checking whether a value is nullish (i.e. `null` or `undefined`) - it weeds out all the values that would produce problems if accessed using a `.` operator (i.e. `null.length`) and also works for things like `0` or `false`